docs/reference/announcements.md

```diff
@@ -4,6 +4,30 @@ title: "Announcements"
 description: "Important announcements including deprecation & removal notices"
 ---
 
+## Camunda 8.5
+
+Release date: 9th of April 2024
+
+End of maintenance: 14th of October 2025
+
+:::info
+
+### Camunda Cloud: New Generation naming scheme
```

Comment:
```suggestion
### Camunda SaaS: New Generation naming scheme
```

AFAIK `SaaS`is the term we should use (compare https://camunda.com/pricing/). But this is also a question for @christinaausley  / DevEx :-)

Reply:
Camunda 8 SaaS/ Camunda 8 Self-Managed is the preference here.